### DISCUSSION TITLE
Pause rendering

### DISCUSSION THREAD

**Discussion by vogloblinsky** [google/model-viewer#4013] [Q&A] [answered]
Hi,

I am using ModelViewer inside a modal. Is it possible to pause rendering when the modal is not displayed ?

Thanks

**Comment by elalish** [comment, marked as answer]
Rendering is automatically paused if the element is not visible on the page (checked via IntersectionObserver), so you should be good to go. It also won't render if nothing has changed, so if you haven't set `autorotate` and you have `interaction-prompt='none'`, it won't re-render even if it's visible.
